2024-07-23 17:23:57,836 INFO    MainThread:26773 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-07-23 17:23:57,836 INFO    MainThread:26773 [wandb_setup.py:_flush():76] Configure stats pid to 26773
2024-07-23 17:23:57,836 INFO    MainThread:26773 [wandb_setup.py:_flush():76] Loading settings from /Users/maxfeucht/.config/wandb/settings
2024-07-23 17:23:57,836 INFO    MainThread:26773 [wandb_setup.py:_flush():76] Loading settings from /Users/maxfeucht/Documents/VU/Diffusion Thesis/VariationalColdDiffusion/wandb/settings
2024-07-23 17:23:57,836 INFO    MainThread:26773 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-07-23 17:23:57,836 INFO    MainThread:26773 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-07-23 17:23:57,836 INFO    MainThread:26773 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': '/Users/maxfeucht/Documents/VU/Diffusion Thesis/VariationalColdDiffusion/main.py', 'program': '/Users/maxfeucht/Documents/VU/Diffusion Thesis/VariationalColdDiffusion/main.py'}
2024-07-23 17:23:57,836 INFO    MainThread:26773 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-07-23 17:23:57,836 INFO    MainThread:26773 [wandb_init.py:_log_setup():527] Logging user logs to /Users/maxfeucht/Documents/VU/Diffusion Thesis/VariationalColdDiffusion/wandb/run-20240723_172357-orjmnmkg/logs/debug.log
2024-07-23 17:23:57,836 INFO    MainThread:26773 [wandb_init.py:_log_setup():528] Logging internal logs to /Users/maxfeucht/Documents/VU/Diffusion Thesis/VariationalColdDiffusion/wandb/run-20240723_172357-orjmnmkg/logs/debug-internal.log
2024-07-23 17:23:57,837 INFO    MainThread:26773 [wandb_init.py:_jupyter_setup():473] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x2ae49a300>
2024-07-23 17:23:57,837 INFO    MainThread:26773 [wandb_init.py:init():567] calling init triggers
2024-07-23 17:23:57,837 INFO    MainThread:26773 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {'timesteps': 6, 'prediction': 'vxt', 'dataset': 'mnist', 'degradation': 'black_blur', 'batch_size': 64, 'dim': 64, 'lr': 0.0001, 'epochs': 3, 'noise_schedule': 'cosine', 'loss_weighting': False, 'var_sampling_step': 1, 'min_t2_step': 1, 'baseline': 'xxxx', 'multiscale_vae': False, 'autoencoder': False, 'kl_annealing': 5000, 'cold_perturb': False, 'vae': False, 'vae_beta': 0.1, 'latent_dim': 32, 'ris_noise': False, 'add_noise': False, 'break_symmetry': False, 'noise_scale': 0.01, 'vae_loc': 'emb', 'vae_inject': 'concat', 'xt_dropout': 0, 'load_checkpoint': False, 'sample_interval': 1, 'n_samples': 60, 'fix_sample': True, 'skip_ema': False, 'model_ema_decay': 0.998, 'cluster': False, 'skip_wandb': False, 'verbose': False, 'multi_gpu': False, 'fid_only': False, 'test_run': False, 'num_downsamples': 2, 'device': 'mps', 'image_size': 28}
2024-07-23 17:23:57,837 INFO    MainThread:26773 [wandb_init.py:init():617] starting backend
2024-07-23 17:23:57,837 INFO    MainThread:26773 [wandb_init.py:init():621] setting up manager
2024-07-23 17:23:57,838 INFO    MainThread:26773 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-07-23 17:23:57,841 INFO    MainThread:26773 [wandb_init.py:init():629] backend started and connected
2024-07-23 17:23:57,851 INFO    MainThread:26773 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-07-23 17:23:57,851 INFO    MainThread:26773 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-07-23 17:23:57,851 INFO    MainThread:26773 [wandb_init.py:init():721] updated telemetry
2024-07-23 17:23:57,875 INFO    MainThread:26773 [wandb_init.py:init():754] communicating run to backend with 90.0 second timeout
2024-07-23 17:23:59,035 INFO    MainThread:26773 [wandb_run.py:_on_init():2344] communicating current version
2024-07-23 17:23:59,455 INFO    MainThread:26773 [wandb_run.py:_on_init():2353] got version response upgrade_message: "wandb version 0.17.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-07-23 17:23:59,455 INFO    MainThread:26773 [wandb_init.py:init():805] starting run threads in backend
2024-07-23 17:24:01,520 INFO    MainThread:26773 [wandb_run.py:_console_start():2323] atexit reg
2024-07-23 17:24:01,520 INFO    MainThread:26773 [wandb_run.py:_redirect():2178] redirect: wrap_raw
2024-07-23 17:24:01,521 INFO    MainThread:26773 [wandb_run.py:_redirect():2243] Wrapping output streams.
2024-07-23 17:24:01,521 INFO    MainThread:26773 [wandb_run.py:_redirect():2268] Redirects installed.
2024-07-23 17:24:01,521 INFO    MainThread:26773 [wandb_init.py:init():848] run started, returning control to user process
2024-07-23 17:24:13,254 INFO    MainThread:26773 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-07-23 17:24:13,255 INFO    MainThread:26773 [wandb_init.py:_pause_backend():438] pausing backend
